[CCR & Snapshot+Restore] Center align states under tabs

sabarasaba opened the pull request:
Addresses remaining work that was discussed https://github.com/elastic/kibana/pull/102507#issuecomment-865120815

I also found a small bug in the ILM app, and since it was such a small fix I ended up doing it here also 🙈 

Related: #100748

<details>
<summary>Screenshots CCR</summary>

#### Follower Indices Tab
![Screenshot 2021-06-24 at 13 09 43](https://user-images.githubusercontent.com/1191206/123253285-9c53b000-d4ed-11eb-9a2e-9600654b6eb8.png)![Screenshot 2021-06-24 at 13 10 02](https://user-images.githubusercontent.com/1191206/123253295-9eb60a00-d4ed-11eb-9dfb-2753cea24ea1.png)![Screenshot 2021-06-24 at 13 09 06](https://user-images.githubusercontent.com/1191206/123253289-9d84dd00-d4ed-11eb-85d6-96809ad6a20c.png)
![Screenshot 2021-06-24 at 13 07 06](https://user-images.githubusercontent.com/1191206/123253294-9e1d7380-d4ed-11eb-8585-e97e090045ab.png)
![Screenshot 2021-06-24 at 13 15 26](https://user-images.githubusercontent.com/1191206/123253878-43d0e280-d4ee-11eb-89e2-22ca71f57acd.png)

#### Auto-follow Patterns Tab
![Screenshot 2021-06-24 at 13 12 05](https://user-images.githubusercontent.com/1191206/123253753-1d12ac00-d4ee-11eb-8888-d1a3d809b30f.png)
![Screenshot 2021-06-24 at 13 11 40](https://user-images.githubusercontent.com/1191206/123253757-1e43d900-d4ee-11eb-9173-035ec843941e.png)
![Screenshot 2021-06-24 at 13 12 30](https://user-images.githubusercontent.com/1191206/123253758-1edc6f80-d4ee-11eb-8a7d-d0fe96132f4d.png)
![Screenshot 2021-06-24 at 13 12 42](https://user-images.githubusercontent.com/1191206/123253759-1f750600-d4ee-11eb-85e8-ac13f0235cfe.png)
![Screenshot 2021-06-24 at 13 14 08](https://user-images.githubusercontent.com/1191206/123253760-200d9c80-d4ee-11eb-8b6c-7c1cf24638ad.png)


</details>


<details>
<summary>Screenshots Snapshot+Restore</summary>

#### Repositories Tab
![Screenshot 2021-06-24 at 12 52 10](https://user-images.githubusercontent.com/1191206/123251366-6ad9e500-d4eb-11eb-80c7-bb7b06b3eafc.png)
![Screenshot 2021-06-24 at 12 52 53](https://user-images.githubusercontent.com/1191206/123251371-6c0b1200-d4eb-11eb-9cc6-297c82673aae.png)
![Screenshot 2021-06-24 at 12 53 40](https://user-images.githubusercontent.com/1191206/123251373-6ca3a880-d4eb-11eb-8f1c-a658e8e7292f.png)
![Screenshot 2021-06-24 at 12 53 57](https://user-images.githubusercontent.com/1191206/123251376-6d3c3f00-d4eb-11eb-8640-fc95a5cbb4f9.png)

#### Policies Tab
![Screenshot 2021-06-24 at 12 56 58](https://user-images.githubusercontent.com/1191206/123251615-b2607100-d4eb-11eb-832f-9d1652d430b3.png)
![Screenshot 2021-06-24 at 12 56 00](https://user-images.githubusercontent.com/1191206/123251621-b3919e00-d4eb-11eb-98f4-5c4117464a49.png)
![Screenshot 2021-06-24 at 12 56 26](https://user-images.githubusercontent.com/1191206/123251624-b42a3480-d4eb-11eb-9804-0221a256804c.png)
![Screenshot 2021-06-24 at 12 56 44](https://user-images.githubusercontent.com/1191206/123251626-b4c2cb00-d4eb-11eb-953f-00bf57da84cf.png)

#### Snapshots Tab
![Screenshot 2021-06-24 at 12 59 17](https://user-images.githubusercontent.com/1191206/123252030-1b47e900-d4ec-11eb-9658-6d00e355e7c6.png)
![Screenshot 2021-06-24 at 12 57 55](https://user-images.githubusercontent.com/1191206/123252031-1c791600-d4ec-11eb-935a-38305b223c80.png)
![Screenshot 2021-06-24 at 12 58 43](https://user-images.githubusercontent.com/1191206/123252033-1d11ac80-d4ec-11eb-84ca-00d54897f6ec.png)
![Screenshot 2021-06-24 at 12 59 02](https://user-images.githubusercontent.com/1191206/123252036-1daa4300-d4ec-11eb-90d3-f0a185e21a7c.png)
![Screenshot 2021-06-24 at 12 59 35](https://user-images.githubusercontent.com/1191206/123252039-1e42d980-d4ec-11eb-8745-8d9cb99215c5.png)
![Screenshot 2021-06-24 at 12 59 58](https://user-images.githubusercontent.com/1191206/123252043-1edb7000-d4ec-11eb-9f6f-bc3aebdb7f96.png)

#### Restore Tab
![Screenshot 2021-06-24 at 13 01 18](https://user-images.githubusercontent.com/1191206/123252294-6a8e1980-d4ec-11eb-94f1-1da75fdc3cc5.png)
![Screenshot 2021-06-24 at 13 00 56](https://user-images.githubusercontent.com/1191206/123252300-6bbf4680-d4ec-11eb-810f-2ce971735bab.png)
![Screenshot 2021-06-24 at 13 01 47](https://user-images.githubusercontent.com/1191206/123252304-6cf07380-d4ec-11eb-9d8d-34d5a3db5f37.png)
![Screenshot 2021-06-24 at 13 02 04](https://user-images.githubusercontent.com/1191206/123252305-6d890a00-d4ec-11eb-9b0c-10c6de25eb88.png)


</details>

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133971/)
* Commit: a14cce658175f44dc8248e3a294891082f54cd26
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Check Types and Public API Docs](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133971/execution/node/428/log)

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @sabarasaba

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133971/","number":"133971","commit":"a14cce658175f44dc8248e3a294891082f54cd26"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133980/)
* Commit: f6ec2d7a4e5574179d73d7b5065a1bb524ba70c2
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Check Types and Public API Docs](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133980/execution/node/428/log)

### History
* :broken_heart: [Build #133971](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133971/) failed a14cce658175f44dc8248e3a294891082f54cd26

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @sabarasaba

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133971/","number":"133971","commit":"a14cce658175f44dc8248e3a294891082f54cd26"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133980/","number":"133980","commit":"f6ec2d7a4e5574179d73d7b5065a1bb524ba70c2"}]}
PIPELINE-->

sabarasaba commented:
@elasticmachine merge upstream

sabarasaba commented on x-pack/plugins/index_lifecycle_management/public/application/sections/policy_table/policy_table.tsx:
```diff
@@ -85,8 +85,10 @@ export const PolicyTable: React.FunctionComponent<Props> = ({
       );
     }
 
+    // Wrapping the actual contents inside a div, otherwise the table layout will get messed up
```
Found this bug while manually testing the apps we migrated, so decided to also patch it up real quick in this PR since it was such a small change. 

https://user-images.githubusercontent.com/1191206/123261155-aded8580-d4f6-11eb-923c-4e85528c3db0.mov

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134000/)
* Commit: 9ba89d717c64b2c6c1f13cbcdbf43dbf61499e47
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Quick Commit Checks](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134000/execution/node/417/log)

### History
* :broken_heart: [Build #133980](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133980/) failed f6ec2d7a4e5574179d73d7b5065a1bb524ba70c2
* :broken_heart: [Build #133971](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133971/) failed a14cce658175f44dc8248e3a294891082f54cd26

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @sabarasaba

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133971/","number":"133971","commit":"a14cce658175f44dc8248e3a294891082f54cd26"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133980/","number":"133980","commit":"f6ec2d7a4e5574179d73d7b5065a1bb524ba70c2"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134000/","number":"134000","commit":"9ba89d717c64b2c6c1f13cbcdbf43dbf61499e47"}]}
PIPELINE-->

sabarasaba commented on x-pack/plugins/cross_cluster_replication/public/app/sections/home/auto_follow_pattern_list/auto_follow_pattern_list.js:
```diff
@@ -170,19 +174,22 @@ export class AutoFollowPatternList extends PureComponent {
-          />
-        </SectionUnauthorized>
+          error={{
+            error: (
```
The `PageError` component doesn't seem to accept a custom error message and I really wanted to re-use it here. We can either do something like this, or modify the component to accept another prop

elasticmachine commented:
Pinging @elastic/kibana-stack-management (Team:Stack Management)

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133992/)
* Commit: a2ed1037d57f01378c8c0348b10d5e7d0e46b074
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-103237/a2ed1037d57f01378c8c0348b10d5e7d0e46b074)
* This comment will update when the build is complete

### Failed CI Steps
* [Lint: eslint](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133992/execution/node/419/log)

### History
* :broken_heart: [Build #134000](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134000/) failed 9ba89d717c64b2c6c1f13cbcdbf43dbf61499e47
* :broken_heart: [Build #133980](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133980/) failed f6ec2d7a4e5574179d73d7b5065a1bb524ba70c2
* :broken_heart: [Build #133971](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133971/) failed a14cce658175f44dc8248e3a294891082f54cd26

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @sabarasaba

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133971/","number":"133971","commit":"a14cce658175f44dc8248e3a294891082f54cd26"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133980/","number":"133980","commit":"f6ec2d7a4e5574179d73d7b5065a1bb524ba70c2"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134000/","number":"134000","commit":"9ba89d717c64b2c6c1f13cbcdbf43dbf61499e47"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133992/","number":"133992","commit":"a2ed1037d57f01378c8c0348b10d5e7d0e46b074"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134018/)
* Commit: 28511e370e98de731d7aa4e8f9f010d89b02d0f8
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-103237/28511e370e98de731d7aa4e8f9f010d89b02d0f8)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/d44f9fe6e6fa85ca411ac9c9e89c7e6711d09753) | [after](https://github.com/elastic/kibana/commit/28511e370e98de731d7aa4e8f9f010d89b02d0f8) | diff |
| --- | --- | --- | --- |
| `crossClusterReplication` | 290.5KB | 290.3KB | -228.0B |
| `indexLifecycleManagement` | 248.6KB | 248.6KB | -33.0B |
| `snapshotRestore` | 458.9KB | 459.6KB | +751.0B |
| total |  |  | +490.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/d44f9fe6e6fa85ca411ac9c9e89c7e6711d09753) | [after](https://github.com/elastic/kibana/commit/28511e370e98de731d7aa4e8f9f010d89b02d0f8) | diff |
| --- | --- | --- | --- |
| `crossClusterReplication` | 24.9KB | 25.1KB | +230.0B |
| `snapshotRestore` | 40.9KB | 41.1KB | +166.0B |
| total |  |  | +396.0B |

### History
* :broken_heart: [Build #133992](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133992/) failed a2ed1037d57f01378c8c0348b10d5e7d0e46b074
* :broken_heart: [Build #134000](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134000/) failed 9ba89d717c64b2c6c1f13cbcdbf43dbf61499e47
* :broken_heart: [Build #133980](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133980/) failed f6ec2d7a4e5574179d73d7b5065a1bb524ba70c2
* :broken_heart: [Build #133971](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133971/) failed a14cce658175f44dc8248e3a294891082f54cd26

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @sabarasaba

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133971/","number":"133971","commit":"a14cce658175f44dc8248e3a294891082f54cd26"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133980/","number":"133980","commit":"f6ec2d7a4e5574179d73d7b5065a1bb524ba70c2"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134000/","number":"134000","commit":"9ba89d717c64b2c6c1f13cbcdbf43dbf61499e47"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133992/","number":"133992","commit":"a2ed1037d57f01378c8c0348b10d5e7d0e46b074"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134018/","number":"134018","commit":"28511e370e98de731d7aa4e8f9f010d89b02d0f8"}]}
PIPELINE-->

alisonelizabeth reviewed (commented):
Thanks so much for working on this @sabarasaba! The screens look great - nice to see all of this coming together and aligning on patterns across our apps.

I noticed one issue with the text rendering outside the error prompt in Snapshot + Restore (although this may be an EUI issue).

<img width="796" alt="Screen Shot 2021-06-24 at 10 29 09 AM" src="https://user-images.githubusercontent.com/5226211/123287561-ce104b00-d4dc-11eb-943a-1d2c9f459b40.png">

[EDITED] I also have a couple concerns around the refactoring made to Snapshot + Restore. First of all, I agree returning early is the better approach and aligns with some of the comments Seb made in https://github.com/elastic/kibana/pull/101548. That said, in a couple places with your current changes (tried to make comments where applicable) I think we lose the privileges check. I _think_ we'd always want this check to happen.

I would also add that Snapshot + Restore is known for having some gaps in test coverage (old issue, but mostly still relevant: https://github.com/elastic/kibana/issues/47595). I would just lean more on the cautious side when refactoring the code and focus on the minimal changes needed to align the UX until we have more robust test coverage.

alisonelizabeth commented:
One more thing 😄 

Do you think this screen should be shown as an error, rather than just the subdued empty prompt? 

<img width="1559" alt="Screenshot" src="https://user-images.githubusercontent.com/1191206/123252036-1daa4300-d4ec-11eb-90d3-f0a185e21a7c.png">

sabarasaba commented:
Thanks for having a look @alisonelizabeth! I think I agree with you, lets better defer the refactoring for later and stick to the little ux changes for now. I've reverted the changes back to using the content var as it was before. I've also made a few more changes as suggested:

* Converted the `snapshots list -> some repositories contains errors` to an actual error overlay
* Wrapped the `PageError` paragraph that shows the error message with an `eui-textBreakWord` class so that if a message is long enough, it wont overflow out of the container

<img width="1427" alt="Screenshot 2021-06-28 at 09 27 55" src="https://user-images.githubusercontent.com/1191206/123591218-5c920e80-d7f4-11eb-8269-3686ecafdee8.png">

sabarasaba commented:
@elasticmachine merge upstream

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134460/)
* Commit: 7e3bc6fc1cf5d72ba8068b86482440d9d09289ca
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-103237/7e3bc6fc1cf5d72ba8068b86482440d9d09289ca)
* This comment will update when the build is complete

### Failed CI Steps
* [Lint: eslint](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134460/execution/node/419/log)

### History
* :green_heart: [Build #134018](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134018/) succeeded 28511e370e98de731d7aa4e8f9f010d89b02d0f8
* :broken_heart: [Build #133992](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133992/) failed a2ed1037d57f01378c8c0348b10d5e7d0e46b074
* :broken_heart: [Build #134000](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134000/) failed 9ba89d717c64b2c6c1f13cbcdbf43dbf61499e47
* :broken_heart: [Build #133980](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133980/) failed f6ec2d7a4e5574179d73d7b5065a1bb524ba70c2
* :broken_heart: [Build #133971](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133971/) failed a14cce658175f44dc8248e3a294891082f54cd26

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @sabarasaba

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133971/","number":"133971","commit":"a14cce658175f44dc8248e3a294891082f54cd26"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133980/","number":"133980","commit":"f6ec2d7a4e5574179d73d7b5065a1bb524ba70c2"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134000/","number":"134000","commit":"9ba89d717c64b2c6c1f13cbcdbf43dbf61499e47"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133992/","number":"133992","commit":"a2ed1037d57f01378c8c0348b10d5e7d0e46b074"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134018/","number":"134018","commit":"28511e370e98de731d7aa4e8f9f010d89b02d0f8"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134460/","number":"134460","commit":"7e3bc6fc1cf5d72ba8068b86482440d9d09289ca"}]}
PIPELINE-->

alisonelizabeth reviewed (approved):
Thanks for making those changes @sabarasaba! Latest LGTM. Looks like there's a small `prettier` error causing CI to fail, but happy to see this merged once that's addressed.

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134939/)
* Commit: 6a54065e8249d5773038cdb2ff87cfed8e649f10
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-103237/6a54065e8249d5773038cdb2ff87cfed8e649f10)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/45dfaac75fcb5d29b5b0fdda3abd3ec9546f7efb) | [after](https://github.com/elastic/kibana/commit/6a54065e8249d5773038cdb2ff87cfed8e649f10) | diff |
| --- | --- | --- | --- |
| `crossClusterReplication` | 290.5KB | 290.3KB | -228.0B |
| `indexLifecycleManagement` | 248.6KB | 248.6KB | -33.0B |
| `snapshotRestore` | 458.9KB | 460.2KB | +1.3KB |
| total |  |  | +1.0KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/45dfaac75fcb5d29b5b0fdda3abd3ec9546f7efb) | [after](https://github.com/elastic/kibana/commit/6a54065e8249d5773038cdb2ff87cfed8e649f10) | diff |
| --- | --- | --- | --- |
| `crossClusterReplication` | 24.9KB | 25.1KB | +230.0B |
| `esUiShared` | 192.5KB | 192.6KB | +27.0B |
| `snapshotRestore` | 40.9KB | 41.1KB | +166.0B |
| total |  |  | +423.0B |

### History
* :broken_heart: [Build #134460](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134460/) failed 7e3bc6fc1cf5d72ba8068b86482440d9d09289ca
* :green_heart: [Build #134018](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134018/) succeeded 28511e370e98de731d7aa4e8f9f010d89b02d0f8
* :broken_heart: [Build #133992](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133992/) failed a2ed1037d57f01378c8c0348b10d5e7d0e46b074
* :broken_heart: [Build #134000](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134000/) failed 9ba89d717c64b2c6c1f13cbcdbf43dbf61499e47
* :broken_heart: [Build #133980](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133980/) failed f6ec2d7a4e5574179d73d7b5065a1bb524ba70c2

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

cc @sabarasaba

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133980/","number":"133980","commit":"f6ec2d7a4e5574179d73d7b5065a1bb524ba70c2"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134000/","number":"134000","commit":"9ba89d717c64b2c6c1f13cbcdbf43dbf61499e47"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/133992/","number":"133992","commit":"a2ed1037d57f01378c8c0348b10d5e7d0e46b074"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134018/","number":"134018","commit":"28511e370e98de731d7aa4e8f9f010d89b02d0f8"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134460/","number":"134460","commit":"7e3bc6fc1cf5d72ba8068b86482440d9d09289ca"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/134939/","number":"134939","commit":"6a54065e8249d5773038cdb2ff87cfed8e649f10"}]}
PIPELINE-->
